app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/EventBadge.kt

```diff
@@ -96,17 +94,9 @@ private fun ConnectRequestBadge(modifier: Modifier = Modifier) {
 
 @Composable
 private fun ConnectPendingRequestBadge(modifier: Modifier = Modifier) {
-    WireSecondaryButton(
-        onClick = { },
-        leadingIcon = {
-            Text(text = stringResource(id = R.string.connection_pending_label))
-        },
-        fillMaxWidth = false,
-        minHeight = dimensions().badgeSmallMinSize.height,
-        minWidth = dimensions().badgeSmallMinSize.width,
-        shape = RoundedCornerShape(size = dimensions().spacing6x),
-        contentPadding = PaddingValues(horizontal = dimensions().spacing6x),
-        textStyle = MaterialTheme.wireTypography.label01
+    WireItemLabel(
+        text = stringResource(id = R.string.connection_pending_label),
+        modifier = modifier
```

Comment:
don't we want to override the `clickable` field to make sure it can't be clicked?
```suggestion
        modifier = modifier.clickable(enabled = false) {}
```

Reply:
The bug happening was that the component was being clickable by itself, so there was a visual effect and a no action taking place. 

If we use this modifier, we'll get rid of the clickable effect, but we will be interfering in the row click again, so we don't need them, since the backing component is a `Text` , without clickable properties by default =D